frontend/src/pages/EditUserPage.tsx

```diff
@@ -146,21 +146,42 @@ const EditUserPage = (props: IEditUserPageProps) => {
             {props => (
               <Form className="userInfo">
                 <Label>IDIR/BCeID</Label>
-                <Input field="username" value={props.values.username} readOnly={true} type="text" />
+                <Input
```

Comment:
You can also get inputs using container.querySelector(`input[name="username"]`);